modules/nixosModules.nix

```diff
@@ -1,11 +1,12 @@
 {
   inputs,
-  hermes,
+  pkgs,
   ...
-}: let
-  std = inputs.nix-std;
-in {
+}: {
   flake.nixosModules = {
-    hermes = import ../../nixosModules/hermes/default.nix {inherit nix-std hermes;};
+    hermes = import ../../nixosModules/hermes/default.nix {
+      inherit (inputs) nix-std; 
+      inherit (pkgs) hermes;
+    };
```

Comment:
I'm not sure if this will receive a `pkgs`, since this is at the top-level "flake" attr, it might need to be system independent. We might actually not be able to default the hermes package here, we might need to just depend on hermes being present in `pkgs` in the actual module.

Reply:
yeah. it needs overlay used by consumer, but i guess that is default.

Reply:
Yeah, but I don't think the default arguments provided by flake parts will give a `pkgs` argument to the modules declared in the `import` list [here](https://github.com/informalsystems/cosmos.nix/blob/main/flake.nix#L17-L36).

Right now the flake doesn't error cause nix is lazy, but I think if someone consumed our flake and used the `nixosModules.hermes` they would get an error that it wasn't called with `pkgs`.

Reply:
that is true. but i fixed overlay to use https://github.com/informalsystems/cosmos.nix/pull/232  as long as overlay used it will work :) but yeah, let people not using overlay to fix non overlay case.  so i think people using modules will use overlay too 99% of time.